_translator-files/po/it/Command-Line-Options.po

```diff
@@ -1,31 +1,31 @@
 # SOME DESCRIPTIVE TITLE
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
+# Giuseppe <dzpex@libero.it>, 2022.
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2022-10-19 10:07+0000\n"
+"Last-Translator: Giuseppe <dzpex@libero.it>\n"
 "Language-Team: LANGUAGE <please@seeGitHubOrWeblate.local>\n"
-"Language: \n"
+"Language: it\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.14.2-dev\n"
 
 #. type: YAML Front Matter: lang
 #: ../wiki/en/Command-Line-Options.md:1
 #, no-wrap
 msgid "en"
-msgstr ""
+msgstr "it"
 
 #. type: YAML Front Matter: layout
 #: ../wiki/en/Command-Line-Options.md:1
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "guida"
```

Comment:
Probably worth a bulk fix and a label on weblate

Reply:
Went to fix that, but "kingu" is not/no longer(?) an admin.
https://hosted.weblate.org/translate/jamulus/client-troubleshooting/en/?checksum=d9c67f8154384e99 is the string anyway.

Reply:
If you change it in one doc, the correction will spread to all others in that language.

Reply:
Just changed it to verify - it does automatically spread.